Brazilian Portuguese Language Support 🇧🇷 (#553)
InnerSourceCommons/innersourcecommons.org: content/pt-br/about/board/_index.md

---
title: "Board & Governance"
type: "board"

> subtitle: A InnerSource Commons é uma organização sem fins lucrativos 501(c)(3) governada por um conjunto de estatutos sociais. O Conselho de Administração estabelece as políticas e nomeia os executivos que definem e executam as políticas. O Conselho é eleito pelos membros anualmente. A InnerSource Commons está incorporada nos Estados Unidos. À medida que a comunidade cresce, esperamos fundar organizações irmãs na União Europeia, América Latina e outras partes do mundo.

Reviewer: subtitle: A InnerSource Commons é uma organização sem fins lucrativos 501(c)(3) regida por um conjunto de estatutos corporativos. O Conselho de Administração define a política e nomeia diretores que definem e executam a política. O Conselho é eleito pelos membros anualmente. A InnerSource Commons está incorporada nos Estados Unidos. À medida que a comunidade cresce, esperamos fundar organizações irmãs na União Europeia, América Latina e outras partes do mundo.

> subtitle: A InnerSource Commons é uma organização sem fins lucrativos 501(c)(3) governada por um conjunto de estatutos sociais. O Conselho de Administração estabelece as políticas e nomeia os executivos que definem e executam as políticas. O Conselho é eleito pelos membros anualmente. A InnerSource Commons está incorporada nos Estados Unidos. À medida que a comunidade cresce, esperamos fundar organizações irmãs na União Europeia, América Latina e outras partes do mundo.

Author: ```suggestion
subtitle: A InnerSource Commons é uma organização sem fins lucrativos 501(c)(3) regida por um conjunto de estatutos corporativos. O Conselho de Administração define a política e nomeia diretores que definem e executam a política. O Conselho é eleito pelos membros anualmente. A InnerSource Commons está incorporada nos Estados Unidos. À medida que a comunidade cresce, esperamos fundar organizações irmãs na União Europeia, América Latina e outras partes do mundo.```

aliases:
    - /board
---

<!--
  ## Instruções para as fotos de perfil:

  As fotos devem ter exatamente 300x300 px, com o rosto centrado na imagem. Armazene os arquivos das fotos na pasta `/static/images/about` no formato JPG ou PNG e mantenha o tamanho do arquivo razoavelmente pequeno (~50KB).

  Observação: Um efeito de escala de cinza será aplicado automaticamente às fotos (por meio de CSS), para deixar a visão geral do Conselho mais coesa.
-->

<div class="section-first container">
    <h2>O Conselho atual:</h2>
</div>

{{< board-member name="Isabel Drost-Fromm" title="Presidente" image="/images/about/Isabel_Drost-Fromm.jpeg" style="section-first" >}}

> Isabel Drost-Fromm é Estrategista de Código Aberto na Europace AG, na Alemanha. Ela é membro da Apache Software Foundation, co-fundadora do Apache Mahout e orientou vários projetos em fase de incubação. Isabel se interessa por tudo relacionado a FOSS (Software de Código Aberto), busca e mineração de texto, com um bom conhecimento em aprendizado de máquina. Seguindo a natureza das pessoas que vivem em Berlim, ela adora receber amigos para uma breve visita. Como resultado, ela co-fundou e ainda é uma das mentes criativas por trás do Berlin Buzzwords, uma conferência de tecnologia sobre busca, escalabilidade e armazenamento.

Reviewer: Isabel Drost-Fromm é Estrategista de Open Source na Europace AG, na Alemanha. Ela é membro da Apache Software Foundation, co-fundadora do Apache Mahout e orientou vários projetos em fase de incubação. Isabel se interessa por tudo relacionado a FOSS, busca e mineração de texto, com um bom conhecimento em aprendizado de máquina. Seguindo a natureza das pessoas que vivem em Berlim, ela adora receber amigos para uma breve visita. Como resultado, ela co-fundou e ainda é uma das mentes criativas por trás do Berlin Buzzwords, uma conferência de tecnologia sobre busca, escalabilidade e armazenamento.

> Isabel Drost-Fromm é Estrategista de Código Aberto na Europace AG, na Alemanha. Ela é membro da Apache Software Foundation, co-fundadora do Apache Mahout e orientou vários projetos em fase de incubação. Isabel se interessa por tudo relacionado a FOSS (Software de Código Aberto), busca e mineração de texto, com um bom conhecimento em aprendizado de máquina. Seguindo a natureza das pessoas que vivem em Berlim, ela adora receber amigos para uma breve visita. Como resultado, ela co-fundou e ainda é uma das mentes criativas por trás do Berlin Buzzwords, uma conferência de tecnologia sobre busca, escalabilidade e armazenamento.

Author: ```suggestion
Isabel Drost-Fromm é Estrategista de Open Source na Europace AG, na Alemanha. Ela é membro da Apache Software Foundation, co-fundadora do Apache Mahout e orientou vários projetos em fase de incubação. Isabel se interessa por tudo relacionado a FOSS, busca e mineração de texto, com um bom conhecimento em aprendizado de máquina. Seguindo a natureza das pessoas que vivem em Berlim, ela adora receber amigos para uma breve visita. Como resultado, ela co-fundou e ainda é uma das mentes criativas por trás do Berlin Buzzwords, uma conferência de tecnologia sobre busca, escalabilidade e armazenamento.```

{{< /board-member >}}

{{< board-member name="Daniel Izquierdo Cortazar" title="Presidente" image="/images/about/Daniel_Izquierdo.jpeg" style="bg-light" >}}

> Daniel Izquierdo Cortazar é pesquisador e um dos fundadores da Bitergia, uma empresa que oferece análises de software para ecossistemas de código aberto. Atualmente, é diretor de dados da Bitergia e concentra-se na qualidade dos dados, pesquisa de novas métricas, análises e estudos de interesse para os clientes da Bitergia por meio de mineração e processamento de dados. Daniel possui um doutorado em engenharia de software de código aberto pela Universidad Rey Juan Carlos, em Madrid, onde se concentrou na análise de padrões de atividade de desenvolvedores com erros na comunidade Mozilla. Daniel faz parte do conselho e é membro da InnerSource Commons.

Reviewer: ```suggestion
Daniel Izquierdo Cortazar é pesquisador e um dos fundadores da Bitergia, uma empresa que oferece análises de software para ecossistemas Open Source. Atualmente, é diretor de dados da Bitergia e concentra-se na qualidade dos dados, pesquisa de novas métricas, análises e estudos de interesse para os clientes da Bitergia por meio de mineração e processamento de dados. Daniel possui um doutorado em engenharia de open source software pela Universidad Rey Juan Carlos, em Madrid, onde se concentrou na análise de padrões de atividade de desenvolvedores com erros na comunidade Mozilla. Daniel faz parte do conselho e é membro da InnerSource Commons.
```

Author: Faça a fusão uma vez! Acho que ficou bom!

{{< /board-member >}}

{{< board-member name="Sebastian Spier" title="Secretário Assistente" image="/images/about/Sebastian_Spier.jpg" >}}
Sebastian Spier é Diretor de Programas de Engenharia na Meltwater, onde trabalha em ferramentas e métodos para melhorar o trabalho diário de mais de 500 engenheiros, removendo atritos sempre que possível. Como gerente de engenharia de diversas equipes de desenvolvimento de software, ele vivenciou o InnerSource como um bloco fundamental para apoiar a colaboração bem-sucedida em equipes distribuídas. Como membro da InnerSource Commons Foundation, ele mantém a coleção de Padrões InnerSource, sempre procurando ajudar os outros a utilizar e aprimorar esses padrões.
{{< /board-member >}}

{{< board-member name="Georg Grütter" image="/images/about/Georg_Grutter.jpeg" style="bg-light" >}}

> Georg Grütter é um evangelista de código social e defensor de desenvolvedores na Bosch.IO. Ele co-fundou e liderou a primeira comunidade InnerSource na Bosch. Georg é um desenvolvedor de software apaixonado com mais de 30 anos de experiência. Anteriormente, ocupou diversos cargos e funções na Bosch, Line Information, Zurich System House e DaimlerChrysler. Georg criou dois projetos de código aberto e também adora fotografia e chocolate.

Reviewer: ```suggestion
Georg Grütter é um evangelista de código social e defensor de desenvolvedores na Bosch.IO. Ele co-fundou e liderou a primeira comunidade InnerSource na Bosch. Georg é um desenvolvedor de software apaixonado com mais de 30 anos de experiência. Anteriormente, ocupou diversos cargos e funções na Bosch, Line Information, Zurich System House e DaimlerChrysler. Georg criou dois projetos open source e também adora fotografia e chocolate.
```

{{< /board-member >}}

{{< board-member name="Yuki Hattori" image="/images/about/Yuki_Hattori.jpg" >}}

> Yuki é um especialista em tecnologia distinto, atualmente atuando como Arquiteto na GitHub, com vasta experiência em tecnologia em nuvem e metodologias DevOps. Ao longo de sua carreira, ele ofereceu orientação valiosa e proficiência para uma infinidade de clientes corporativos, ajudando-os a aprimorar seus sistemas e processos. A área de especialização de Yuki é a transformação InnerSource e ele estabeleceu uma comunidade próspera no Japão que se envolveu com figuras proeminentes e líderes de pensamento da indúst

Reviewer: ```suggestion
é um distinto especialista em tecnologia que atualmente atua como arquiteto no GitHub, com vasta experiência em tecnologia em nuvem e metodologias DevOps. Ao longo de sua carreira, ele ofereceu orientação valiosa e proficiência para uma infinidade de clientes corporativos, ajudando-os a aprimorar seus sistemas e processos. A área de especialização de Yuki é a transformação InnerSource e ele estabeleceu uma comunidade próspera no Japão que se envolveu com figuras proeminentes e líderes de pensamento da indústria. 
```

> 

Reviewer: ```suggestion
```

> ria. Como defensor apaixonado do InnerSource, Yuki traduziu uma vasta gama de materiais e organizou inúmeros workshops e apresentações em vídeo para disseminar a conscientização e a compreensão do assunto. Ele se orgulha imensamente de servir como exemplo para outras regiões e continua comprometido em empregar sua experiência e conhecimento para beneficiar outros membros da comunidade.

Reviewer: ```suggestion
Como defensor apaixonado do InnerSource, Yuki traduziu uma vasta gama de materiais e organizou inúmeros workshops e apresentações em vídeo para disseminar a conscientização e a compreensão do assunto. Ele se orgulha imensamente de servir como exemplo para outras regiões e continua comprometido em empregar sua experiência e conhecimento para beneficiar outros membros da comunidade.
```

{{< board-member name="Matt Cobby" image="/images/about/Matt_Cobby.jpg" >}}
Matt Cobby é Diretor de Engenharia na Deloitte, onde se especializa em ajudar equipes com suas práticas de engenharia, experiência do desenvolvedor e engenharia de plataforma e produtos. Como membro da InnerSource Foundation, ele conduziu um programa InnerSource em grande escala e agora busca inspirar equipes a buscar os muitos benefícios de um estilo de colaboração de código aberto dentro de suas próprias organizações. Com mais de 18 anos de experiência em transformação no Reino Unido, Europa e Austrália, ele tem paixão por orientar engenheiros e é um apoiador ativo de comunidades técnicas e locais que inspiram as pessoas.
{{< /board-member >}}

<div class="section-first container">
    <h2>Histórico:</h2>
</div>

<div class="container">
  <p>
    Essas pessoas incríveis já fizeram parte do nosso Conselho em anos anteriores e as listamos aqui para mostrar nossa gratidão por seu trabalho. Agradecemos sua dedicação à InnerSource e seu apoio ativo à InnerSource Commons Foundation!
  </p>
</div>

{{< board-member name="Silona Bonewald" title="Tesoureira" image="/images/about/Silona_Bonewald.jpeg" >}}

> Silona Bonewald é Diretora Executiva do IEEE SA OPEN, uma plataforma abrangente que oferece à comunidade de código aberto opções econômicas para desenvolver e validar seus projetos. Anteriormente, ela foi vice-presidente de arquitetura de comunidade na Hyperledger, um esforço colaborativo global de código aberto sediado na Linux Foundation, onde integrou líderes financeiros, bancários, de IoT, cadeias de suprimentos e manufatura. Outras realizações profissionais notáveis incluem pioneirismo no processo InnerSource enquanto estava no PayPal, criação de um site de comércio eletrônico inovador e compatível com Six Sigma para a Siemens AG, e criação de uma arquitetura de sistema de gerenciamento de conteúdo (CMS) internacional para a Ubisoft.

Reviewer: ```suggestion
Silona Bonewald é Diretora Executiva do IEEE SA OPEN, uma plataforma abrangente que oferece à comunidade Open Source opções econômicas para desenvolver e validar seus projetos. Anteriormente, ela foi vice-presidente de arquitetura de comunidade na Hyperledger, um esforço colaborativo global de código aberto sediado na Linux Foundation, onde integrou líderes financeiros, bancários, de IoT, cadeias de suprimentos e manufatura. Outras realizações profissionais notáveis incluem pioneirismo no processo InnerSource enquanto estava no PayPal, criação de um site de comércio eletrônico inovador e compatível com Six Sigma para a Siemens AG, e criação de uma arquitetura de sistema de gerenciamento de conteúdo (CMS) internacional para a Ubisoft.
```

{{< /board-member >}}

{{< board-member name="Maximilian Capraro" title="Tesoureiro Assistente" image="/

images/about/Maximilian_Capraro.jpg" >}}

> Maximilian Capraro é pesquisador e candidato a doutorado no Open Source Research Group na Friedrich-Alexander-University Erlangen-Nürnberg. Ele trabalhou em projetos de pesquisa de InnerSource com diversas empresas, incluindo Black Duck Software, Continental e várias unidades de negócios da Siemens. Ele desenvolveu o método de medição de fluxo de patch (para medir a colaboração no desenvolvimento de software) e o primeiro framework de classificação para projetos e programas InnerSource. Suas áreas de pesquisa incluem quantificação da colaboração no desenvolvimento de software, governança de InnerSource, incluindo aspectos legais e tributários, e muitos outros tópicos relacionados a InnerSource, código aberto e engenharia de software.

Reviewer: ```suggestion
Maximilian Capraro é pesquisador e candidato a doutorado no Open Source Research Group na Friedrich-Alexander-University Erlangen-Nürnberg. Ele trabalhou em projetos de pesquisa de InnerSource com diversas empresas, incluindo Black Duck Software, Continental e várias unidades de negócios da Siemens. Ele desenvolveu o método de medição de fluxo de patch (para medir a colaboração no desenvolvimento de software) e o primeiro framework de classificação para projetos e programas InnerSource. Suas áreas de pesquisa incluem quantificação da colaboração no desenvolvimento de software, governança de InnerSource, incluindo aspectos legais e tributários, e muitos outros tópicos relacionados a InnerSource, Open Source e engenharia de software.
```

{{< /board-member >}}

{{< board-member name="Danese Cooper" image="/images/about/Danese_Cooper.jpg" >}}

> Danese Cooper é a fundadora da InnerSource Commons. Ela é uma defensora de longa data do código aberto, tendo atuado anteriormente como chefe de software de código aberto do PayPal, CTO da Wikimedia Foundation, evangelista-chefe de código aberto da Sun e diretora sênior de estratégias de código aberto da Intel. Danese também foi a primeira presidente da Node.js Foundation. Ela se concentra em criar comunidades de código aberto saudáveis e já integrou os conselhos da Drupal Association, da Open Source Initiative e da Open Source Hardware Association. Ela também aconselhou a Mozilla e a Apache Software Foundation. Danese também dirige uma bem-sucedida consultoria de código aberto que conta com clientes como a Fundação Bill & Melinda Gates, o SETI Institute, a Harris e a Numenta. Ela costuma tricotar durante reuniões.

Reviewer: ```suggestion
Danese Cooper é a fundadora da InnerSource Commons. Ela é uma Open Source advocate de longa data, tendo atuado anteriormente como Head de open source software do PayPal, CTO da Wikimedia Foundation, evangelista-chefe de open surce da Sun e diretora sênior de estratégias de open source da Intel. Danese também foi a primeira presidente da Node.js Foundation. Ela se concentra em criar comunidades open source saudáveis e já integrou os conselhos da Drupal Association, da Open Source Initiative e da Open Source Hardware Association. Ela também é consultora da Mozilla e da Apache Software Foundation. Danese também dirige uma bem-sucedida consultoria de open source que conta com clientes como a Fundação Bill & Melinda Gates, o SETI Institute, a Harris e a Numenta. Ela é conhecida por tricotar durante as reuniões.
```

{{< /board-member >}}

{{< board-member name="Johannes Tigges" image="/images/about/Johannes_Tigges.png" >}}

> Johannes é co-criador do InnerSource Commons Learning Path e trabalhou na introdução do InnerSource na HERE Technologies, uma empresa líder em mapas digitais e localização. Atualmente, ele oferece coaching e consultoria em Produtividade de Engenharia, Estratégia de Código Aberto e InnerSource e tópicos de OSPO, além de arquitetura de soluções em nuvem. Ele também atua como palestrante e oferece treinamentos técnicos sobre os temas mencionados acima. Ele apresenta seu trabalho e pensamentos em conferências do setor, como o FOSDEM ou a InnerSource Commons Summit, e é ativo em comunidades de código aberto há muito tempo. Com formação em ciência da computação e sociologia das organizações, ele possui uma perspectiva única sobre organizações de engenharia de software. Em funções anteriores, ele também trabalhou na introdução de Integração e Entrega Contínua em grandes instituições de pesquisa, em funções de DevOps para plataformas baseadas em nuvem de grande porte e desenvolveu software no domínio de telecomunicações.

Reviewer: ```suggestion
Johannes é co-criador do InnerSource Commons Learning Path e trabalhou na introdução do InnerSource na HERE Technologies, uma empresa líder em mapas digitais e localização. Atualmente, ele oferece coaching e consultoria em Produtividade de Engenharia, Estratégia de Open Source e InnerSource e tópicos de OSPO, além de arquitetura de soluções em nuvem. Ele também atua como palestrante e oferece treinamentos técnicos sobre os temas mencionados acima. Ele apresenta seu trabalho e pensamentos em conferências do setor, como o FOSDEM ou a InnerSource Commons Summit, e é ativo em comunidades open source há muito tempo. Com formação em ciência da computação e sociologia das organizações, ele possui uma perspectiva única sobre organizações de engenharia de software. Em funções anteriores, ele também trabalhou na introdução de Integração e Entrega Contínua em grandes instituições de pesquisa, em funções de DevOps para plataformas baseadas em nuvem de grande porte e desenvolveu software no domínio de telecomunicações.
```

{{< /board-member >}}

{{< board-member name="Jacob Green" image="/images/about/Jacob_Green.jpg" >}}

> Jacob Green, fundador da Mosslabs.io, é um artista de código aberto, estrategista e construtor de sistemas que

Reviewer: ```suggestion
Jacob Green, fundador da Mosslabs.io, é um artista de open source, estrategista e construtor de sistemas que
```

>  trabalha para expandir o impacto e a aplicação do código aberto na sociedade globalmente. Vindo de Baltimore, é apaixonado por código aberto em cidades e universidades, e trabalha para conectar instituições por meio do código aberto. Antes da Mosslabs.io, Jacob foi diretor de iniciativas estratégicas na LTNGlobal, construindo uma rede global de transporte de vídeo única do zero e a partir de princípios fundamentais.

Reviewer: ```suggestion
 trabalha para expandir o impacto e a aplicação de open source na sociedade globalmente. Vindo de Baltimore, é apaixonado por open source em cidades e universidades, e trabalha para conectar instituições por meio de open source. Antes da Mosslabs.io, Jacob foi diretor de iniciativas estratégicas na LTNGlobal, construindo uma rede global de transporte de vídeo única do zero e a partir de princípios fundamentais.
```
